doc/lsp.txt

```diff
@@ -272,8 +272,29 @@ server and to set the language server options.  For example: >
 <
 							*lsp-options*
 Some of the LSP plugin features can be enabled or disabled by using the
-LspOptionsSet() function. This function accepts a dictionary argument with the
-following optional items:
+LspOptionsSet() function.
+
+The options are divided into two categories:
+
+1. |lsp-options-func| Options that changes functionality
+2. |lsp-options-style| Options that changes highlight and text
+
+This function accepts a dictionary argument with options.
+
+For example, to disable the automatic placement of signs for the LSP
+diagnostic messages, you can add the following line to your .vimrc file: >
+
+	call LspOptionsSet({'autoHighlightDiags': v:false})
+<
+The LspOptionsGet() function returns a |Dict| of all the LSP plugin options,
+To get a particular option value you can use the following: >
+
+	echo LspOptionsGet()['autoHighlightDiags']
+<
+
+All options are listed below:
+
+							*lsp-options-func*
```

Comment:
This split of options might not make sense? I think the list of options is growing quite large though.

Reply:
I am thinking of removing the highlight group related options.  The user can override the pre-defined highlight groups by defining them before sourcing the plugin.

Reply:
Yes, that makes perfect sense.

Reply:
I'll remove the commit that reorders the docs then.